calisphere/templates/calisphere/objectViewer/object-tiles-and-overlays.html

```diff
@@ -8,9 +8,29 @@
     {% endif %}
   >
     {% if item.reference_image_md5 or item.contentFile.format|lower == 'file' %}
+      <!-- {{item}} -->
       <div class="obj__icon-container">
         {% if item.reference_image_md5 %}
-          <img class="obj__simple-image" src="{{ thumbnailUrl }}clip/500x500/{{ item.reference_image_md5 }}" alt="{{ item.title.0 }}">
+          <img class="obj__simple-image" src="{{ thumbnailUrl }}clip/500x500/{{ item.reference_image_md5 }}" alt="{{ item.title.0 }}" width={{item.reference_image_dimensions.0}} height={{item.reference_image_dimensions.1}} style="width: 
+              {% if item.reference_image_dimensions.0 > item.reference_image_dimensions.1 %}
+                {% if item.reference_image_dimensions.0|add:0 > 500 %}
+                  500px
+                {% else %}
+                  {{ item.reference_image_dimensions.0 }}
+                {% endif %}
+              {% else %}
+                auto
+              {% endif %}; 
+            height: 
+              {% if item.reference_image_dimensions.1 > item.reference_image_dimensions.0 %}
+                {% if item.reference_image_dimensions.1|add:0 > 500 %}
+                  500px
+                {% else %}
+                  {{ item.reference_image_dimensions.1 }}
+                {% endif %}
+              {% else %}
+                auto
+              {% endif %};">
```

Comment:
for a square image, it will be `auto` and `auto` ?

Reply:
thanks @tingletech - for a square image, neither dimension should be auto. resolved this to explicitly define both dimensions in case of square.